### PR TITLE
added `comment_update` command and simplified `comment_create` usage when replying to existing comment

### PR DESCRIPTION
**Pull request by osilkin98**
## Changes

Suppose we have the following:
claim with `claim_id: fa39ff913ceb021b03acd5acc5eed17365ffc856`
comment with `comment_id: c14c8e8af1d40f09a63beadda2a37d6a0edce70c`

### Current Behavior 
In order to reply to a comment, the user needs to provide its id to the `parent_id` 
param, *and* the `claim_id` of that comment: 

```sh
$ lbrynet comment create "What a thought provoking comment." --claim_id=fa39ff913ceb021b03acd5acc5eed17365ffc856 --parent_id=c14c8e8af1d40f09a63beadda2a37d6a0edce70c
```

### New Behavior
The user only needs the `comment_id` of the comment they're replying to:
```sh
$ lbrynet comment create "Much simpler" --parent_id=c14c8e8af1d40f09a63beadda2a37d6a0edce70c
```

### CODE REVIEW

File: scripts/generate_json_api.py [hunk 1]
```diff
@@ -369,6 +404,11 @@ class Examples(CommandTestCase):
 
         # abandon all the things
 
+        await r(
+            'Abandon a comment',
+            'comment', 'abandon', comment['comment_id']
+        )
```
Comment (eukreign) [inline scripts/generate_json_api.py]: Didn't this comment already get abandoned earlier?

**Comment by osilkin98** [comment]
No, it’s a different one that I kept as reference for other examples and didn’t need to rerun the API for abandon twice

File: scripts/generate_json_api.py (hunk 1 above)
Comment (osilkin98) [reply scripts/generate_json_api.py]: different one

**Comment by lyoshenka** [comment]
@osilkin98 if this has breaking API changes, please document them. lex can help explain the right way to do that

**Comment by eukreign** [comment]
@lyoshenka this doesn't have any breaking API changes

**Pull request merged**